pykusto/client.py

```diff
@@ -207,42 +270,50 @@ def __getattr__(self, name: str) -> Any:
         """
         return self[name]
 
-    def get_table(self) -> KQL:
-        result = KQL(', '.join(self.tables))
-        if '*' in result or ',' in result:
-            result = KQL('union ' + result)
-        return result
+    @staticmethod
+    def static_is_union(*table_names: str) -> bool:
```

Comment:
maybe move this to ItemFetcher?

Reply:
This method is Kusto-specific, while `ItemFetcher` is not